mteb/abstasks/AbsTask.py

```diff
@@ -224,10 +228,42 @@ def load_data(self, **kwargs):
         """
         if self.data_loaded:
             return
-        self.dataset = datasets.load_dataset(**self.metadata.dataset)  # type: ignore
+        if self.is_multilingual:
+            if self.fast_loading:
+                self.fast_load()
+            else:
+                self.dataset = {}
+                for lang in self.hf_subsets:
+                    self.dataset[lang] = datasets.load_dataset(
+                        name=lang,
+                        **self.metadata.dataset,
```

Comment:
I believe we can use this in all cases. (multilingual or not) (is the is_multilingual even required?)

```suggestion
                for hf_subset in self.hf_subsets:
                    self.dataset[hf_subset] = datasets.load_dataset(
                        name= hf_subset,
                        **self.metadata.dataset,
```
As it can also be something other than language (see brightretrieval)